### ISSUE TITLE
Error handling in *beginRenderPass* and *beginComputePass* is underspecified

### ISSUE THREAD

**Issue by jimblandy** [gpuweb/gpuweb#2874]
The *beginRenderPass* algorithm says that if the valid usage requirements are unsatisfied, implementations should "generate a validation error and stop." This leaves unspecified what value is returned, what state the command encoder is in, whether the render pass is valid, and so on. The *beginComputePass* algorithm has the same ambiguity.

I believe both cases should mark the pass and command encoder as invalid, and return the pass.

cc @toji

**Comment by kainino0x** [comment]
Resolved in #2873? I didn't check these particular ones were handled but I think they were.

**Comment by toji** [comment]
Yes, #2873 fixes it. I just managed to push the PR before this issue was filed so they didn't get explicitly linked. 😄
